rosdep/python.yaml

```
@@ -7104,6 +7104,11 @@ python3-nose-yanc:
   debian: [python3-nose-yanc]
   openembedded: [python3-nose-yanc@meta-ros-common]
   ubuntu: [python3-nose-yanc]
+python3-numba:
+  debian: [python3-numba]
+  ubuntu: 
```

Every time I use the web editor it is a mistake.

Fix trailing whitespace.

```suggestion
  ubuntu:
```